src/Slices/Resource/UI/ResourcesPage/Components/RequiresTableWithData.tsx

```diff
@@ -11,35 +10,33 @@ interface Props {
 }
 
 export const RequiresTableWithData: React.FC<Props> = ({ id, deps }) => {
-  const { queryResolver } = useContext(DependencyContext);
+  const { data, isSuccess, isError, error, refetch } = useGetResourceDetails().useContinuous(id);
 
-  const [data] = queryResolver.useContinuous<"GetResourceDetails">({
-    kind: "GetResourceDetails",
-    id,
-  });
+  if (isError) {
+    <ErrorView
+      message={words("error.general")(error?.message)}
```

Comment:
Why do we need the safe check on the error.message here, but not in the ResourcePage/Page.tsx?
```TS
 if (isError) {
    return <ErrorView message={error.message} ariaLabel="ResourcesPage-Error" retry={refetch} />;
  }
```